install/install.md

```diff
@@ -229,9 +239,10 @@ bash /rapids/utils/stop-jupyter.sh
 <div id="pip"></div>
 
 ## **pip**
-Beginning with the release of 23.04: cuDF, dask-cuDF, cuML, cuGraph, RMM, and RAFT CUDA 11 pip packages are available on the NVIDIA Index.
+Beginning with the release of 23.06: cuDF, dask-cuDF, cuML, cuGraph, cuSpatial, RMM, and RAFT CUDA 11 and CUDA 12 pip packages are available on the NVIDIA Index.
```

Comment:
I re-ordered this sentence a bit. The "Beginning with..." made sense when pip packages were new, but not so much now that they're stable.

I googled around a bit and it seems like "NVIDIA Python Package Index" is the name that is being used more broadly, not "NVIDIA Index."
```suggestion
RAPIDS 23.06 pip packages of cuDF, dask-cuDF, cuML, cuGraph, cuSpatial, RMM, and RAFT are available for CUDA 11 and CUDA 12 on the NVIDIA Python Package Index.
```
Also, do we need to list cuCIM? Are we missing anything else?

Reply:
cuCIM is on pypi proper, this should be correct list for what's on pypi.nvidia.com unless we want to include ptxcompiler or something. I wanted to only include items on the selector though. Which makes me realize I need to add a button to the selector for `dask-cudf`.